src/com/hrabrov/introduction_to_java/algorithmization/sorting/Helper.java

```diff
@@ -0,0 +1,35 @@
+package com.hrabrov.introduction_to_java.algorithmization.sorting;
+
+import com.hrabrov.introduction_to_java.algorithmization.sorting.task08.Fraction;
+
+public class Helper {
+    public static String fractionsToString(Fraction[] fractions) {
+        String stringWithFractions = "";
+        for (int i = 0; i < fractions.length; i++) {
+            stringWithFractions += fractions[i].numerator + "/" + fractions[i].denominator + " ";
+        }
+        return stringWithFractions;
+    }
+
+    public static String arrayToString(int[] array) {
+        String stringForStorageArrayValues = "";
+
+        for (int i = 0; i < array.length; i++) {
+            stringForStorageArrayValues += array[i] + " ";
+        }
+
```

Comment:
please, write code at the same style. If you choose not to add blank lines at the first methods, then remove them from all another methods